spark/src/test/scala/geotrellis/spark/io/index/zcurve/ZSpaceTimeKeyIndexSpec.scala

```diff
@@ -93,7 +93,7 @@ class ZSpaceTimeKeySpec extends FunSpec with Matchers{
     }
 
     it("generates indexes by string pattern") {
-      val pattern = "YM" //by months
+      val pattern = "YMM" //by months
```

Comment:
Before: `2015 of Jan` equals to `20151`, `2015 of Dec` equals to `201512`
After: `2015 of Jan` equals to `201501`, `2015 of Dec` equals to `201512`